app/controllers/users.js

```diff
@@ -7,6 +7,7 @@ const avatars = require('./avatars').all();
 const jwt = require('jsonwebtoken');
 const bcrypt = require('bcryptjs');
 require('dotenv').config();
+const sendgridgMail = require('@sendgrid/mail');
```

Comment:
`sendgridMail` is better, instead of `sendgridgMail`

Reply:
Thank you Fortune. I am on it now.